### PR TITLE
Issue 67 scope name

### PR DESCRIPTION
**Pull request by evtimo**

### CODE REVIEW

File: src/main/resources/config/defaults-storage-pravega.json [hunk 1]
```diff
@@ -31,26 +33,15 @@
 			"sndBuf": 0,
 			"tcpNoDelay": true,
 			"timeoutMillis": 0,
-			// The flag to enable the load through SSL/TLS.
```
Comment (veronikaKochugova) [inline src/main/resources/config/defaults-storage-pravega.json]: Why did you delete the comments?

**Review by veronikaKochugova — APPROVED** [review]

File: src/main/resources/config/defaults-storage-pravega.json (hunk 1 above)
Comment (evtimo) [reply src/main/resources/config/defaults-storage-pravega.json]: Error in json format during compilation locally
Travis doesn't check it?
